paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -65,7 +65,17 @@ internal class PaymentSheetPlaygroundViewModel(
     val state = MutableStateFlow<PlaygroundState?>(null)
     val flowControllerState = MutableStateFlow<FlowControllerState?>(null)
     val customerSheetState = MutableStateFlow<CustomerSheetState?>(null)
-    val customerAdapter = MutableStateFlow<CustomerAdapter?>(null)
+    val customerAdapter = MutableStateFlow(
+        CustomerAdapter.create(
+            context = getApplication(),
+            customerEphemeralKeyProvider = {
+                throw NotImplementedError("Nope!")
```

Comment:
Is this just a modeling problem in the playground? Why do we need the not implemented error?